modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -0,0 +1,75 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="configuring_bios_for_du_hosts_{context}"]
+= Configuring BIOS for distributed unit bare-metal hosts
+
+Distributed unit (DU) hosts require certain BIOS settings to be configured before the host can be provisioned.
+
+.Procedure
+. Set the *UEFI/BIOS Boot Mode* to `UEFI` and ensure that the same setting is applied in the `BareMetalHost` CRD.
```

Comment:
I'd expand CRD. Also, will your user need any other context to start this procedure in the right place?

Reply:
Really, this task of BIOS config is outside of the scope of this information. The context is that the BIOS must be configured before you install OCP. It's up to the customer to prepare the hosts for the OCP install. This is a bare minimum guidance for what needs to be configured. It's also dependant on the specific hardware.